packages/core/src/mappers/renderer.ts

```diff
@@ -114,7 +115,7 @@ const isRequired = (
   rootSchema: JsonSchema
 ): boolean => {
   const pathSegments = schemaPath.split('/');
-  const lastSegment = pathSegments[pathSegments.length - 1];
+  let lastSegment = pathSegments[pathSegments.length - 1];
```

Comment:
Let's just decode in place
```suggestion
  const lastSegment = decode(pathSegments[pathSegments.length - 1]);
```

Reply:
Done 👌